zadanie3/src/main/java/gra_w_kosci.java

```diff
@@ -0,0 +1,121 @@
+import java.util.Scanner;
+import java.lang.Exception;
+
+/**
+ * W grze w kości bierze udział dwóch graczy o nazwach: pierwszy, drugi. Każdemu graczowi przypada 5
+ * tur, w każdej turze, gracz wykonuje maksymalnie 10 rzutów dwoma kośćmi jednocześnie. Tury graczy
+ * odbywają się naprzemiennie tzn. grę zaczyna gracz pierwszy, po turze następuje tura gracza
+ * drugiego, następna jest tura gracza pierwszego itd. Możliwe wyniki rzutu jedna kością to jedna z
+ * następujących cyfr {1, 2, 3, 4, 5, 6}. Grę zawsze rozpoczyna gracz ”pierwszy”. Podczas każdej tury
+ * gracz zbiera punkty według zasad opisanych poniżej. Suma punktów każdego gracza przed
+ * rozpoczęciem gry wynosi zero. Wygrywa gracz, który zbierze mniejsza ilość punktów.
+ *
+ * @author Kamil Best
+ */
+class graWKosci {
+    private static final int rzutMin = 1;
+    private static final int rzutMax = 6;
+    private static final int iloscTur = 5;
+
+    /**
+     * Sprawdzenie poprawnosci danych wejsciowych.
+     * Wyrzucenie wyjatku w przypadku blednych danych.
+     *
+     * @param rzut - ilosc oczek wyrzuconych na jednej kostce (1-6)
+     * @throws Exception - wyjatek w przypadku podania jako argument liczby spoza zakresu (1-6)
+     */
+    private void sprawdzPoprawnoscRzutu(int rzut) throws Exception {
+        if (rzut < rzutMin || rzut > rzutMax)
+            throw new Exception("Bledne dane, mozesz uzyc liczb z przedzialu 1-6");
+    }
+
+    /**
+     * Obsluga pojedynczej tury aktualnego gracza.
+     * Pobranie ilosci oczek w obu kostkach, sprawdzenie poprawnosci rzutu, jezeli zle dane, wyrzucenie wyjatki i prosba
+     * o ponowne podanie.
+     * Sumowanie punktow, informowanie o ilosci punktow oraz ewentualnej przegranej lub wygranej w turze.
+     *
+     * @param g - obiekt Klasy Gracz zawierajacy imie gracza
+     * @return suma_pkt - zwraca ilosc uzyskanych przez gracza punktow w danej turze
+     */
+    private int tura(Gracz g) {
+        int numer_rzutu = 1;
+        int pierwsza_kostka, druga_kostka;
+        int suma_oczek = 0, suma_pkt = 0;
+        Scanner rzut = new Scanner(System.in);
+        while (numer_rzutu <= 10) {
+            System.out.println("\nGraczu " + g.zwrocImie() + " podaj ilosc oczek w pierwszej kostce (1-6):");
+            pierwsza_kostka = rzut.nextInt();
+            System.out.println("\nGraczu " + g.zwrocImie() + " podaj ilosc oczek w drugiej kostce (1-6):");
+            druga_kostka = rzut.nextInt();
+            try {
+                sprawdzPoprawnoscRzutu(pierwsza_kostka);
+            } catch (Exception e) {
+                e.printStackTrace();
+                System.out.println("\nNieprawidlowy rzut graczu " + g.zwrocImie() + " podaj ilosc oczek w pierwszej kostce jeszcze raz(1-6):");
+                pierwsza_kostka = rzut.nextInt();
```

Comment:
Teraz jest tak, że jeżeli poda liczbę >6 lub <1, to musi podać jeszcze raz aż do skutku..